src/tests/GC/Scenarios/BaseFinal/basefinal.cs

```diff
@@ -90,7 +90,7 @@ public CreateObj()
         {
 
             mv_Obj = new BaseFinal();
-            //Printing GetTotalMemory is commented out so that the test can build on RedHawk
+            //Printing GetTotalMemory is commented out so that the test can build on NativeAOT
```

Comment:
This is probably a very old comment referring to NativeAOT without a real corelib. We could delete the comment and uncomment the line.